src/commands/nsec3hash.rs

```diff
@@ -40,6 +44,69 @@ pub struct Nsec3Hash {
     name: Name<Vec<u8>>,
 }
 
+const LDNS_HELP: &str = "\
+ldns-nsec3-hash [OPTIONS] <domain name>
+  prints the NSEC3 hash of the given domain name
+
+  -a <algorithm> hashing algorithm number
+  -t <number>    iterations
+  -s <string>    salt in hex\
+";
+
+impl LdnsCommand for Nsec3Hash {
+    const HELP: &'static str = LDNS_HELP;
+
+    fn parse_ldns() -> Result<Self, Error> {
+        let mut algorithm = Nsec3HashAlg::SHA1;
+        let mut iterations = 1;
+        let mut salt = Nsec3Salt::empty();
+        let mut name = None;
+
+        let mut parser = lexopt::Parser::from_env();
+
+        while let Some(arg) = parser.next()? {
+            match arg {
+                Arg::Short('a') => {
+                    let val = parser.value()?;
+                    algorithm =
+                        Self::parse_os_with("algorithm (-a)", &val, Nsec3Hash::parse_nsec_alg)?;
```

Comment:
Readability? We could just do `-a`, I don't really mind.

Reply:
I would go for just `-X` and ideally also find a way to simplify this case of `Arg::Short('X') => { ... "-X" }` as its error prone, in testing this I already accidentally once did `Arg::Short('X') => { ... "-Y" }`...

Reply:
I'm mainly in favour of the shorter text to avoid more duplication, but that's also why I'd be interested in seeing (not necessarily now) if we can automate this duplication to avoid accidentally using different values when they should be the same. For me we can merge this PR without resolving this.